ios/Podfile

```diff
@@ -4,7 +4,7 @@ require_relative '../node_modules/@react-native-community/cli-platform-ios/nativ
 # Edge addition to disable advertising ID collection:
 $RNFirebaseAnalyticsWithoutAdIdSupport = true
 
-platform :ios, '13.0'
+platform :ios, '13.0' # min_ios_version_supported
```

Comment:
We can't use `min_ios_version_supported` it points to a react native variable (currently 12.4) and rnzcash requires 13.